src/transports/smart_protocol.c

```diff
@@ -521,7 +521,7 @@ int git_smart__download_pack(
 	int error = 0;
 	struct network_packetsize_payload npp = {0};
 
-	// TODO
+	/* TODO */
```

Comment:
Hey @ethomson, this one feels a bit out of place. Todo what?

Reply:
Haha, sorry, my bad.  I usually mark things with a `// TODO` because that ridiculous comment style should jump out at me in a PR.  :trollface:  Usually I make sure that I address everything and make sure that all the TODOs are pulled out when I clean up my commit series before I send a PR.  Obviously I missed this one this time, oops.

The comment needs to go and the code surrounding it should be happy and not have anything that needs attention.